test/input-range/input-range.spec.jsx

```diff
@@ -255,6 +255,30 @@ describe('InputRange', () => {
     component.detach();
   });
 
+  it('notifies the parent component when dragging starts', () => {
+    const onChange = jasmine.createSpy('onChange').and.callFake(value => component.setProps({ value }));
+    const onChangeStart = jasmine.createSpy('onChangeStart');
+    const jsx = (
+  	<InputRange
```

Comment:
Could you please fix the indentation of this JSX block? At the moment, I think there are 3 spaces; instead, there should be 6. :)